src/NTDll.Tests/NTDllFacts.cs

```diff
@@ -64,4 +64,27 @@ public unsafe void NtQueryInformationProcess_Test()
             Assert.Equal(NTSTATUS.Code.STATUS_SUCCESS, result.Value);
         }
     }
+
+    [Fact]
+    public void RtlGetVersion_OSVERSIONINFOEX_Test()
+    {
+        var versionInfoEx = Kernel32.OSVERSIONINFOEX.Create();
+
+        Assert.Equal(NTSTATUS.Code.STATUS_SUCCESS, RtlGetVersion(ref versionInfoEx).Value);
+        Assert.Equal(2, versionInfoEx.dwPlatformId); // VER_PLATFORM_WIN32_NT
+        Assert.NotEqual(0, versionInfoEx.dwMajorVersion);
+        Assert.NotEqual(0, versionInfoEx.dwBuildNumber);
+        Assert.True(Enum.IsDefined(typeof(Kernel32.OS_TYPE), versionInfoEx.wProductType), $"Unexpected OS_TYPE value: 0x{(int)versionInfoEx.wProductType:X}");
+    }
+
+    [Fact]
+    public unsafe void RtlGetVersion_OSVERSIONINFO_Test()
+    {
+        var versionInfo = Kernel32.OSVERSIONINFO.Create();
+
+        Assert.Equal(NTSTATUS.Code.STATUS_SUCCESS, RtlGetVersion((Kernel32.OSVERSIONINFOEX*)&versionInfo).Value);
```

Comment:
I wonder whether this is a case where helpers could come in handy. For e.g., in NtDll.Helpers.cs (see Kernel32.Helpers.cs for example), perhaps add a public overload `RtlGetVersion(OSVERSIONINFO*)` and "hide" this type of "reinterpret_cast" type casting in there (and let the users/callers of this API get a clean experience) ? 

Another related thought - if you do add a helper, then would it make sense to use `RtlGetVersion(OSVERSIONINFO)` as the `DllImport` method, and `RtlGetVersion(OSVERSIONINFOEX)` as the helper? This would make the code for the helper itself a bit cleaner (the cast would read like `(OSVERSIONINFO*)&osVersionInfoEx`, which would guarantee that the pointer is within valid bounds and that this fact is statically verifiable) and would also follow the pattern in the OS itself (`GetVersion` and `RtlGetVersion` are defined in terms of `LPOSVERSIONINFO`, and when the `dwOSVersionInfoSize` indicates that the payload is actually an `LPOSVERSIONINFOEX` then additional fields are filled in and returned.)